extract-usage-from-readme.sh

```diff
@@ -0,0 +1,22 @@
+#! /usr/bin/env bash
+# This file is part of git-big-picture
+#
+# Copyright (C) 2021 Sebastian Pipping <sebastian@pipping.org>
+#
+# git-big-picture is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# git-big-picture is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with git-big-picture.  If not, see <http://www.gnu.org/licenses/>.
+
+set -e
+set -o pipefail
+
+fgrep -A55 usage: README.rst | sed 's,^    ,,'
```

Comment:
Hardcoding 55 as the length of the help output seems hard to maintain.  Perhaps you could do something like this?

```
awk '/usage:/ { print; flag = 1; next } /Usage Examples/ { flag = 0 } flag { print }' README.rst
```

Reply:
I agree that this approach would probably require fewer updates to file `extract-usage-from-readme.sh` in the future. I have applied that idea now, could you have another look?

Reply:
Looks good!

Reply:
Okay great! Let's see how happy @esc is with the latest version.